index.bs

```diff
@@ -1,11 +1,62 @@
-<pre class='metadata'>
-Title: Web Extensions
-Shortname: webextensions
-Level: 1
-Status: w3c/CG-DRAFT
-Group: WECG
-URL: https://w3c.github.io/webextensions
-Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
-Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
-Abstract: [Placeholder] Abstract.
-</pre>
+<pre class='metadata'>
+Title: Web Extensions
+Shortname: webextensions
+Level: 1
+Status: w3c/CG-DRAFT
+Group: WECG
+URL: https://w3c.github.io/webextensions
+Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
+Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
+Abstract: [Placeholder] Abstract.
+Markup Shorthands: markdown yes
+</pre>
+
+# Manifest
+
+A WebExtension must have a manifest file at its root directory.
```

Comment:
Should we explain the basic idea of how a web extension is loaded into a browser to make `its root directory` have its context?

Reply:
I’d argue that the spec needs a separate section outside of the manifest definition to define this, and the fact that it can either be a directory or a zip for example.

Reply:
We should define the directory structure in a separate section which would cover the locale directories. However, per this CG’s charter, we have decided the package format (zip, app, etc) are out of scope.

Reply:
We can drop the aspect of location of manifest file in this section and define the directory structure later (as suggested by Timothy).

Reply:
I'm a little torn. On one hand it doesn't matter where the manifest file itself is located, but on the other it _does_ contain information about filesystem pathing for the other resources it mentions. For the moment I'm okay with the current copy.

Reply:
Ok so for the sake of merging this quickly, I’ll let it there (we can certainly move this sentence in another, more appropriate section later).